presto-main/src/main/java/com/facebook/presto/sql/SqlEnvironmentConfig.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.sql;
+
+import io.airlift.configuration.Config;
+
+public class SqlEnvironmentConfig
```

Comment:
We usually add tests for config classes like this one. For example, see how `TestNodeMemoryConfig` tests defaults and explicit configs for `NodeMemoryConfig`.

Reply:
Would it make sense to add a test class now, or would that be better in a later commit once the default is determined?